include/ui/veles_mainwindow.h

```diff
@@ -71,7 +71,7 @@ class VelesMainWindow : public MainWindowWithDetachableDockWidgets {
   void createMenus();
   void createDb();
   void createFileBlob(QString);
-  void createHexEditTab(QString, dbif::ObjectHandle);
+  DockWidget* createHexEditTab(QString, dbif::ObjectHandle);
```

Comment:
I'm missing how is this supposed to work. I can only find 2 places there createHexEditTab is used, and in both of them nothing is done with return value.